R/print.bayesfactor_parameters.R

```diff
@@ -10,7 +10,7 @@ print.bayesfactor_parameters <- function(x, digits = 2, log = FALSE, ...) {
 
   BFE$BF <- .format_big_small(BFE$BF, digits = digits)
 
-  colnames(BFE)[colnames(BFE) == "BF"] <- "Bayes Factor"
+  colnames(BFE)[colnames(BFE) == "BF"] <- "BF"
```

Comment:
I think we don't need code line at all in this case?

Reply:
actually, yes